boards/arm/stm32g071b_disco/stm32g071b_disco_defconfig

```diff
@@ -0,0 +1,44 @@
+# Copyright 2021 The Chromium OS Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_STM32G0X=y
+CONFIG_SOC_STM32G071XX=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
```

Comment:
Due to overhead implied, we advise not to enable by default on boards with less than 64K SRAM.

Reply:
I'll remove it.